docs/elements.md

```diff
@@ -865,25 +865,25 @@ Element to arrange given blocks of code or text that are displayed initially in
+| `source-header`       | string                                          | "Drag from here"               | The text that appears at the start of the source area.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `solution-header`     | string                                          | "Construct your solution here" | The text that appears at the start of the solution area.                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `solution-placement`  | "right" or "bottom"                             | "right"                        | `right` shows the source and solution areas aligned side-by-side. `bottom` shows the solution area below the source area.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `partial-credit`      | "none" or "lcs"                                 | Depends on `grading-method`    | For the `dag`, `ordered`, and `ranking` grading methods, you may specify `none` for no partial credit or `lcs` for partial credit based on the LCS edit-distance from the student solution to some correct solution. Note that `partial-credit` defaults to `lcs` for the `dag` and `ranking` grading method, and to `none` for the `ordered` grading method for backward compatibility. For the other grading methods, using this property is not yet supported. Grading method `unordered` will always assign partial credit. |
```

Comment:
you should just remove the sentence 'For the other grading methods, using this property is not yet supported.' Its a bit awkward sounding, now that there is only one 'other' grading method, and we already make it clear what happens with that one in the next sentence.

Reply:
Fixed in [04ad4e5](https://github.com/PrairieLearn/PrairieLearn/pull/9292/commits/04ad4e56c0b854239b8f727954053ead55b70cc0).